packages/dids/src/did.methods/did.key.manager.ts

```diff
@@ -0,0 +1,52 @@
+import { driver } from 'did-method-key';
+import { IDidManager, DIDResponse } from '../interface/i.did.manager';
+import {Wallet} from '@docknetwork/wallet-sdk-core/lib/modules/wallet';
+
+export class DidKeyManager  implements IDidManager{
+  private static instance: IDidManager;
+
+  private readonly wallet;
+
+  private constructor() {
+    this.wallet = Wallet.getInstance();
+  }
+  async createDID(options = {}): Promise<DIDResponse> {
+    const didDocument = await driver().generate();
+    return this.saveDID(didDocument)
+  }
+
+
+  public static getInstance(): IDidManager {
+    if (!DidKeyManager.instance) {
+      DidKeyManager.instance = new DidKeyManager();
+    }
+    return DidKeyManager.instance;
+  }
+
+
+  async saveDID(didDocument: any): Promise<DIDResponse> {
+    const doc = await this.wallet.add({
+      type: 'DID',
```

Comment:
this isnt the correct document format for a DID resolution. it should be like: https://w3c-ccg.github.io/universal-wallet-interop-spec/#DIDResolutionResponse

Reply:
Yeah. "content" property has the actual document

Reply:
which isnt correct